### PR TITLE
CKV_AZURE_70

### PR DESCRIPTION
**Pull request by hadarpl**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/function_app#https_only
Ensure that Function apps is only accessible over HTTPS

### CODE REVIEW

File: checkov/terraform/checks/resource/azure/FunctionAppsAccessibleOverHttps.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from checkov.common.models.enums import CheckCategories
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+
+
+class FunctionAppsAccessibleOverHttps(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure Disks are encrypted at rest"
```
Comment (nimrodkor) [inline checkov/terraform/checks/resource/azure/FunctionAppsAccessibleOverHttps.py:7]: @hadarpl Name is not related I believe

File: checkov/terraform/checks/resource/azure/FunctionAppsAccessibleOverHttps.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        id = "CKV_AZURE_51"
+        supported_resources = ['azurerm_function_app']
+        categories = [CheckCategories.NETWORKING]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def get_inspected_key(self):
+        return 'https_only'
+
+    def get_expected_value(self):
+        return True
```
Comment (nimrodkor) [inline checkov/terraform/checks/resource/azure/FunctionAppsAccessibleOverHttps.py:17]: @hadarpl This is by default, no need to override

**Review by nimrodkor — APPROVED** [review]

File: checkov/terraform/checks/resource/azure/FunctionAppsAccessibleOverHttps.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+from checkov.common.models.enums import CheckCategories
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+
+
+class FunctionAppsAccessibleOverHttps(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure Disks are encrypted at rest"
+        id = "CKV_AZURE_51"
```
Comment (nimrodkor) [inline checkov/terraform/checks/resource/azure/FunctionAppsAccessibleOverHttps.py:8]: @hadarpl This causes the UT to fail as well - this is not the correct ID

Comment (hadarpl) [reply checkov/terraform/checks/resource/azure/FunctionAppsAccessibleOverHttps.py]: Fixed!

**Pull request merged**